doc/source/serve/monitoring.md

```diff
@@ -234,23 +234,52 @@ In addition to the standard Python logger, Serve supports custom logging. Custom
+logging for Ray Serve. Pass a dictionary or object of [LoggingConfig](../serve/api/doc/ray.serve.schema.LoggingConfig.rst)
+to the `logging_config` argument of `serve.run` or `@serve.deployment`.
 
 #### Configure logging format
-You can configure the JSON logging format by passing `encoding=JSON` to `logging_config` argument of `@serve.deployment`. For example:
+You can configure the JSON logging format by passing `encoding=JSON` to `logging_config`
+argument in `serve.run` or `@serve.deployment`
 
+::::{tab-set}
+
+:::{tab-item} `serve.run`
+```{literalinclude} doc_code/monitoring/logging_config.py
+:start-after: __serve_run_json_start__
+:end-before: __serve_run_json_end__
+:language: python
+```
+:::
+
+:::{tab-item} `@serve.deployment`
 ```{literalinclude} doc_code/monitoring/logging_config.py
-:start-after: __json_start__
-:end-before: __json_end__
+:start-after: __deployment_json_start__
+:end-before: __deployment_json_end__
 :language: python
 ```
-In the `Model` log file, you should see the following:
+:::
+
+::::
+
+In the replica `Model` log file, you should see the following:
 
 ```
-{"levelname": "INFO", "asctime": "2023-12-07 12:59:45,271", "deployment": "Model", "replica": "default#Model#PUGBSJ", "request_id": "8d316c3b-4c9f-4769-8080-b6867ca46d1e", "route": "/", "application": "default", "message": "replica.py:719 - Started executing request 8d316c3b-4c9f-4769-8080-b6867ca46d1e"}
-{"levelname": "INFO", "asctime": "2023-12-07 12:59:45,271", "deployment": "Model", "replica": "default#Model#PUGBSJ", "request_id": "8d316c3b-4c9f-4769-8080-b6867ca46d1e", "route": "/", "application": "default", "message": "replica.py:745 - __CALL__ OK 0.1ms"}
+# cat `ls /tmp/ray/session_latest/logs/serve/replica_default_Model_*`
+
+{"levelname": "INFO", "asctime": "2024-02-27 10:36:08,908", "deployment": "default_Model", "replica": "rdofcrh4", "message": "replica.py:855 - Started initializing replica."}
+{"levelname": "INFO", "asctime": "2024-02-27 10:36:08,908", "deployment": "default_Model", "replica": "rdofcrh4", "message": "replica.py:877 - Finished initializing replica."}
+{"levelname": "INFO", "asctime": "2024-02-27 10:36:10,127", "deployment": "default_Model", "replica": "rdofcrh4", "request_id": "f4f4b3c0-1cca-4424-9002-c887d7858525", "route": "/", "application": "default", "message": "replica.py:1068 - Started executing request to method '__call__'."}
+{"levelname": "INFO", "asctime": "2024-02-27 10:36:10,127", "deployment": "default_Model", "replica": "rdofcrh4", "request_id": "f4f4b3c0-1cca-4424-9002-c887d7858525", "route": "/", "application": "default", "message": "replica.py:373 - __CALL__ OK 0.6ms"}
 ```
 
+:::{note}
+`RAY_SERVE_ENABLE_JSON_LOGGING=1` environment variable is getting deprecated in the
+next release. If you have the need to override the global default for Serve logging
+format without setting `logging_config`, you can set `RAY_SERVE_LOG_ENCODING=JSON`
+environment variable to achieve the same.
```

Comment:
```suggestion
The `RAY_SERVE_ENABLE_JSON_LOGGING=1` environment variable is getting deprecated in the
next release. To enable JSON logging globally, use `RAY_SERVE_LOG_ENCODING=JSON`.
```
nits in wording, but was `RAY_SERVE_ENABLE_JSON_LOGGING=1` previously documented? As far as I know this was not really exposed as a public API (in which case, we shouldn't need to mention anything here)

Reply:
Yes, it was in Ray 2.8 or before https://docs.ray.io/en/releases-2.8.1/serve/monitoring.html#json-logging-format And the user got confused it being dropped from the doc despite still in the code. I will remove it from both code and the doc after 2.10 is cut.